executor/executor.go

```diff
@@ -172,7 +172,6 @@ func (executor *Executor) createServiceForFunction(meta *metav1.ObjectMeta) (*fs
 	}
 
 	if fsvcErr != nil {
-		fsvcErr = errors.Wrap(fsvcErr, fmt.Sprintf("[%v] Error creating service for function", meta.Name))
```

Comment:
Why remove this line? It's useful for troubleshooting if the log contains metadata.

Reply:
It was same information being printed by the caller and the called function - this making overall message to user pretty verbose. Shall we just log meta?

Reply:
I'm doubt by removing this line can help us to eliminate verbose log entirelly cause this line just adds some extra information here. In fact, I think we should improve the log here. 

As you can see, the newdeploy print and return same error log back to caller.
https://github.com/fission/fission/blob/1dbb4f4fb73ed1a3be97f87b059c8a5e1ebc9536/executor/newdeploy/newdeploymgr.go#L301-L304
And for poolmanager, some of errors are returned directly.
https://github.com/fission/fission/blob/1dbb4f4fb73ed1a3be97f87b059c8a5e1ebc9536/executor/executor.go#L159-L167

How about we leave this line as the same, and open other PR to address verbose log issue here?